### ISSUE TITLE
[Question] Traefik and KeyBox (jetty/websocket application)

### ISSUE THREAD

**Issue by giopas** [containous/traefik#850]
Hi,

I am trying to configure KeyBox [1] with Traefik in order to have a reverse proxy to it.

The application, served by jetty, runs on port 9880.

I have therefore create this simple configuration:

```toml
[backends]
[backends.keybox]
        [backends.keybox.servers.server1]
            url = "http://127.0.0.1:9880"

[frontends]
[frontends.keybox]
     backend = "keybox"
     entrypoints = ["http"]
         [frontends.keybox.routes.keybox_1]
         rule = "Host:keybox.mydomain.com"
```

The problem is that in such way I am forwarded to the following URL: http://127.0.0.1:58000/login.action

The same reverse proxy can be done with Apache 2.4.x with the following Virtual Host:

```
<VirtualHost *:80>
 	ServerName keybox.mydomain.com
 	ProxyRequests Off
<Proxy *>
 	#Order deny,allow
 	Require all granted
</Proxy>
<LocationMatch "/admin/(terms.*)">
 	ProxyPass ws://localhost:9880/admin/$1
 	ProxyPassReverse ws://localhost:9880/admin/$1
</LocationMatch>
 	ProxyPass / http://localhost:9880/
 	ProxyPassReverse / http://localhost:9880/
 	ProxyRequests Off
	ProxyPreserveHost On
 	ProxyStatus On
</VirtualHost>
```
What am I missing here?
If the configuration bloc is fine, can it be related to this issue? [2]

Thanks for your advises!

giopas

[1]: http://sshkeybox.com/
[2]: https://github.com/skavanagh/KeyBox/issues/181

**Comment by giopas** [comment]
Hello,

anyone ( @errm @emilevauge maybe?) could help on this please?

At first sight it could seems similar to issue #920 but it is not a matter of /.

Thank you,

giopas

**Comment by ldez** [comment]
Closed by #1638, in the version 1.3.0-rc3.
